src/main/resources/db/changelog/db.changelog-1.0.xml

```diff
@@ -407,4 +407,13 @@
         </addColumn>
     </changeSet>
 
+    <changeSet id="2022-01-04-add-new-email-types" author="mikhuttu">
+        <insert tableName="email_type">
+            <column name="name" value="AUTHORISATION_EXPIRY"/>
```

Comment:
Insertion of AUTHORISATION_EXPIRY could be removed from this pull request but then yet another changeSet has to be added in OPHAKTKEH-132. That might be useful though in case someone wants to work on that while having this PR still in review phase (and not on top of this one).